doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/getting-started.ipynb

```diff
@@ -271,102 +280,34 @@
-    "# NAME                                             READY   STATUS    RESTARTS   AGE\n",
-    "# raycluster-autoscaler-head-xxxxx                 2/2     Running   0          XXs\n",
-    "# raycluster-autoscaler-worker-small-group-yyyyy   1/1     Running   0          XXs\n",
-    "# raycluster-autoscaler-worker-small-group-zzzzz   1/1     Running   0          XXs "
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "222c64f4",
-   "metadata": {},
-   "source": [
-    "To get a summary of your cluster's status, run `ray status` on your cluster's Ray head node."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "6fa373aa",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Substitute your head pod's name in place of \"raycluster-autoscaler-head-xxxxx\"\n",
-    "! kubectl exec raycluster-autoscaler-head-xxxxx -it -c ray-head -- ray status\n",
+    "To set up your Ray Cluster for Ray Jobs submission, we just need to make sure that the Ray Jobs port is visible to the client.\n",
+    "Ray listens for Job requests through the head pod's Dashboard server.\n",
     "\n",
-    "# ======== Autoscaler status: 2022-07-21 xxxxxxxxxx ========\n",
-    "# ...."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b56e07a3",
-   "metadata": {},
-   "source": [
-    "Alternatively, to examine the full autoscaling logs, fetch the stdout of the Ray head pod's autoscaler sidecar:"
+    "First, we need to find the location of the Ray head node. The KubeRay operator configures a [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) targeting the Ray head pod. This service allows us to interact with Ray clusters without directly executing commands in the Ray container. To identify the Ray head service for our example cluster, run:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "4ec93304",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# This command gets the last 20 lines of autoscaler logs.\n",
-    "\n",
-    "# Substitute your head pod's name in place of \"raycluster-autoscaler-head-xxxxx\"\n",
-    "! kubectl logs raycluster-autoscaler-head-xxxxx -c autoscaler | tail -n 20\n",
-    "\n",
-    "# ======== Autoscaler status: 2022-07-21 xxxxxxxxxx ========\n",
-    "# ..."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "40d0a503",
-   "metadata": {},
-   "source": [
-    "### The Ray head service"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "c6e257f2",
-   "metadata": {},
-   "source": [
-    "The KubeRay operator configures a [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) targeting the Ray head pod. This service allows us to interact with Ray clusters without directly executing commands in the Ray container. To identify the Ray head service for our example cluster, run"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
```

Comment:
sry again for the ipynb format --
might help to refresh the notebook and re-push to get rid of the stray diff
(on the other hand, this diff might be immaterial)

Reply:
Doesn't seem to have changed the content. Let's just worry about this later when we move away from ipynb.

Reply:
okidoke